sass/layout/_headers.scss

```diff
@@ -180,6 +180,12 @@
     .disabled {
         color: #999999;
     }
+
+    .pinned-posts-button {
+        svg {
+            fill: rgba(var(--center-channel-color-rgb), 0.6);
+        }
+    }
```

Comment:
```suggestion
```

Reply:
Apologies @zeirash, but it looks like the class `.pinned-posts-button` is actually no longer used. The channel header had some work done but it looks like this `changeCSS` line was inadvertently left behind.

@asaadmahmood, do you concur?

Reply:
so does that mean issue #16029 is no longer valid @deanwhillier ?

Reply:
Probably. We can close this ticket then.